devops/jobs/CloudFlareHitRate.groovy

```diff
@@ -0,0 +1,75 @@
+/*
+
+    Variables without defaults are marked (required)
+
+    Variables consumed for this job:
+        * NOTIFY_ON_FAILURE: (Required - email address)
+        * SECURE_GIT_CREDENTIALS: (Required - jenkins name of git credentials)
+        * ZONE_ID: (Required - The CloudFlare Zone ID)
+        * AUTH_KEY: (Required - Authentication key for the account that would make API calls)
+        * EMAIL: (Required - Email of the account for making API calls)
+
+*/
+package devops.jobs
+
+
+import static org.edx.jenkins.dsl.DevopsTasks.common_parameters
+
+
+class CloudflareHitRate {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey('ZONE_ID') : "Zone id of domain"
```

Comment:
The assertion errors should be more clear about the error. eg. `Required ZONE_ID setting missing from configuration.`

Similarly for the two assertions below as well.